lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -1108,6 +1111,23 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
       // Make sure that if operand is generic that its primary archetypes match
       // the function context.
       checkLegalType(I->getFunction(), operand.get(), I);
+
+      // If we are not in OSSA, our operand constraint should be none if we have
```

Comment:
"should be none if we have"...
replace with
"should be invalid for a type dependent operand" (or Optional::None)